Interfaces/API/DiracAdmin.py

```diff
@@ -262,9 +262,11 @@ def getSiteMaskLogging(self, site=None, printOutput=False):
             gLogger.notice(stup)
           gLogger.notice(' ')
       elif isinstance(sitesLogging, list):
-        result = [(sl[1], sl[3], sl[4]) for sl in sitesLogging]
+	sitesLoggingList = [(sl[1], sl[3], sl[4]) for sl in sitesLogging]
+	for siteLog in sitesLoggingList:
+	  gLogger.notice(siteLog)
```

Comment:
?
```suggestion
	  sitesLoggingList = [(sl[1], sl[3], sl[4]) for sl in sitesLogging]
	  for siteLog in sitesLoggingList:
	    gLogger.notice(siteLog)
```

Reply:
I introduced by mistake some tabs instead of spaces...